src/test/java/net/snowflake/client/jdbc/ConnectionLatestIT.java

```diff
@@ -78,6 +78,8 @@ public class ConnectionLatestIT extends BaseJDBCTest {
 
   private boolean defaultState;
 
+  private final String ENABLE_BOUNCYCASTLE_PROVIDER_JVM = "net.snowflake.jdbc.enableBouncyCastle";
```

Comment:
let's reuse the constant src/main

Reply:
can we use the const from `SessionUtilKeyPair `?

Reply:
@sfc-gh-dprzybysz I will have to make SessionUtilKeyPair public in order to be able to access it from outside of that package, and make that variable static. 
Or I move this constant to SecurityUtil and use that in both ConnectionLatestIT and SessionUtilKeyPair. I think the second option would be better.

Reply:
we can go with second option 👍